src/platform/forms/save-in-progress/FormStartControls.jsx

```diff
@@ -64,20 +63,23 @@ class FormStartControls extends React.Component {
   };
 
   render() {
+    const { formConfig } = this.props.routes[1];
+    const startNewAppMessage =
+      formConfig.startNewAppMessage || 'Start a new application';
```

Comment:
Because it looks like we'll be using these in several modules, let's either:
- Set the default messages direct in the `formConfig` object when we load it up, or
- Pull the magic strings into constants for reuse

Reply:
If I remember correctly, the generator for the form system creates the `savedFormMessages` property on the `formConfig` object so unless it's added to the generator, adding it to all existing `formConfig`'s wouldn't cover future forms. So I'll opt for a constants file unless we can add that to the generator.

Reply:
> Set the default messages direct in the `formConfig` object when we load it up

I meant after it was loaded into the library, not in the actual definition of it in the `config/form.js` module, so the generator wouldn't play a role in it, I think.

That said, using a shared constant is great! I don't think I personally have a preference of the two options I presented. :+1: